repos/resource_git_credential.go

```diff
@@ -2,15 +2,23 @@ package repos
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"strconv"
 	"strings"
 
+	"github.com/databricks/databricks-sdk-go/apierr"
 	"github.com/databricks/databricks-sdk-go/service/workspace"
 	"github.com/databricks/terraform-provider-databricks/common"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+func isOnlyOneGitCredentialError(err error) bool {
```

Comment:
did you want this error or the old one?
```suggestion
func isOnlyOneGitCredentialForProviderError(err error) bool {
```